cpp/tests/c_api/mg_eigenvector_centrality_test.c

```diff
@@ -106,16 +107,16 @@ int test_eigenvector_centrality(const cugraph_resource_handle_t* handle)
 
   // Pagerank wants store_transposed = TRUE
   return generic_eigenvector_centrality_test(handle,
-                               h_src,
-                               h_dst,
-                               h_wgt,
-                               h_result,
-                               num_vertices,
-                               num_edges,
-                               TRUE,
-                               alpha,
-                               epsilon,
-                               max_iterations);
+                                             h_src,
+                                             h_dst,
+                                             h_wgt,
+                                             h_result,
+                                             num_vertices,
+                                             num_edges,
+                                             TRUE,
+                                             alpha,
+                                             epsilon,
+                                             max_iterations);
```

Comment:
Isn't the updates in this file from the eigenvector C API PR?

Reply:
I had to modify the call to `create_mg_test_graph` and `create_test_graph` (the SG version) to add the is_symmetric parameter.  This required me to change all of the C unit tests.  Not quite sure why the indentation changed.  Perhaps an upstream change in the clang format parameters?